sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala

```diff
@@ -81,6 +81,90 @@ class JsonFunctionsSuite extends QueryTest with SharedSparkSession {
       expected)
   }
 
+  test("SPARK-42782: Port get_json_object's tests from the Apache Hive project") {
```

Comment:
Why don't you name this test title like "SPARK-42782: Hive compatibility check for get_json_object"

Reply:
Do you prefer "[SPARK-42782](https://issues.apache.org/jira/browse/SPARK-42782): Hive compatibility check for get_json_object"?

Reply:
Yeah but I don't mind if you don't prefer that. Will leave it to you.

Reply:
Renamed it.